Build fail with NETWORKING OFF

Looks like the usual problem of trying to use distributed stuff when it should be disabled
```
/home/biddisco/src/hpx/libs/init_runtime/src/hpx_init.cpp: In function ‘int hpx::disconnect(double, double, hpx::error_code&)’:
/home/biddisco/src/hpx/libs/init_runtime/src/hpx_init.cpp:991:17: error: ‘get_runtime_distributed’ was not declared in this scope; did you mean ‘runtime_distributed’?
  991 |                 get_runtime_distributed().get_runtime_support_lva());
      |                 ^~~~~~~~~~~~~~~~~~~~~~~
      |                 runtime_distributed
/home/biddisco/src/hpx/libs/init_runtime/src/hpx_init.cpp: In function ‘void hpx::terminate()’:
/home/biddisco/src/hpx/libs/init_runtime/src/hpx_init.cpp:1022:17: error: ‘get_runtime_distributed’ was not declared in this scope; did you mean ‘runtime_distributed’?
 1022 |                 get_runtime_distributed().get_runtime_support_lva());
      |                 ^~~~~~~~~~~~~~~~~~~~~~~
      |                 runtime_distributed
```

--- a/libs/init_runtime/src/hpx_init.cpp
+++ b/libs/init_runtime/src/hpx_init.cpp
@@ -35,6 +35,7 @@
 #include <hpx/program_options/variables_map.hpp>
 #include <hpx/resource_partitioner/partitioner.hpp>
 #include <hpx/runtime/naming_fwd.hpp>
+#include <hpx/runtime/runtime_fwd.hpp>
 #include <hpx/runtime_local/config_entry.hpp>
 #include <hpx/runtime_local/custom_exception_info.hpp>
 #include <hpx/runtime_local/debugging.hpp>

--- a/libs/performance_counters/src/counters.cpp
+++ b/libs/performance_counters/src/counters.cpp
@@ -22,6 +22,7 @@
 #include <hpx/runtime/actions/continuation.hpp>
 #include <hpx/runtime/agas/interface.hpp>
 #include <hpx/runtime/components/stubs/runtime_support.hpp>
+#include <hpx/runtime/runtime_fwd.hpp>
 #include <hpx/runtime_distributed.hpp>
 #include <hpx/runtime_local/get_num_localities.hpp>
 #include <hpx/runtime_local/thread_pool_helpers.hpp>

--- a/src/runtime/agas/addressing_service.cpp
+++ b/src/runtime/agas/addressing_service.cpp
@@ -43,6 +43,7 @@
 #include <hpx/runtime/agas/symbol_namespace.hpp>
 #include <hpx/runtime/find_localities.hpp>
 #include <hpx/runtime/find_here.hpp>
+#include <hpx/runtime/runtime_fwd.hpp>
 #include <hpx/async_base/launch_policy.hpp>
 #include <hpx/runtime/naming/split_gid.hpp>
 #include <hpx/serialization/serialize.hpp>

--- a/src/runtime/components/server/runtime_support_server.cpp
+++ b/src/runtime/components/server/runtime_support_server.cpp
@@ -37,6 +37,7 @@
 #include <hpx/runtime/find_localities.hpp>
 #include <hpx/runtime/naming/resolver_client.hpp>
 #include <hpx/runtime/naming/unmanaged.hpp>
+#include <hpx/runtime/runtime_fwd.hpp>
 #include <hpx/serialization/serialize.hpp>
 #include <hpx/serialization/vector.hpp>
 #include <hpx/runtime_local/shutdown_function.hpp>

--- a/src/runtime_distributed.cpp
+++ b/src/runtime_distributed.cpp
@@ -43,6 +43,7 @@
 #include <hpx/runtime/naming/resolver_client.hpp>
 #include <hpx/runtime/parcelset/parcelhandler.hpp>
 #include <hpx/runtime/parcelset_fwd.hpp>
+#include <hpx/runtime/runtime_fwd.hpp>
 #include <hpx/runtime_local/shutdown_function.hpp>
 #include <hpx/runtime_local/startup_function.hpp>
 #include <hpx/runtime_local/thread_hooks.hpp>

